database/migrations/2021_09_27_133459_add_uxresearch_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUxresearchTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('uxresearch', function (Blueprint $table) {
```

Comment:
let's call this `circle_metadata` instead please (& model name "CircleMetadata")

Reply:
updated